aws/amazon-chime-sdk-js
# Do not try to set groupId  from cache for null device

**Issue #1568 :**

**Description of changes:**
Currently we requires to call list device API before calling choosing device API. Otherwise, the device cache is not populated and the error `Device cache is not populated. Please make sure to call list devices first` will be thrown. However, this does not make sense in the case of calling null device. 
Update the logic when choosing null device to just set groupId to ''  instead of looking the groupId from device cache.

**Testing**

1. Have you successfully run `npm run build:release` locally? Yes
2. How did you test these changes? Add unit tests to cover this case.
    1. In the meeting demo, open console to delete the device cache (`delete app.audioVideo.deviceController.deviceInfoCache`).
    2. Call `app.audioVideo.chooseAudioInputDevice(null)`
    3. Verify that there is no error message log.
3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it? Yes, meeting demo.
4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved? N/A
5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved? N/A


By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
